src/bundle/Resources/translations/messages.en.xliff

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:jms="urn:jms:translation" version="1.2">
+  <file source-language="en" target-language="en" datatype="plaintext" original="not.available">
+    <header>
+      <tool tool-id="JMSTranslationBundle" tool-name="JMSTranslationBundle" tool-version="1.1.0-DEV"/>
+      <note>The source node in most cases contains the sample message as written by the developer. If it looks like a dot-delimitted string such as "form.label.firstname", then the developer has not provided a default message.</note>
+    </header>
+    <body>
+      <trans-unit id="956a777d05ee6c9e537bbd06c94cbaece1846af8" resname="content_forms.relation.location_type.browseDUPA">
```

Comment:
```suggestion
      <trans-unit id="956a777d05ee6c9e537bbd06c94cbaece1846af8" resname="content_forms.relation.location_type.browse">
```
      
here and below ;)

Reply:
Is it right time to introduce adamwojs/php-cs-fixer-bad-language rule to ibexa/code-style ? :D